### DISCUSSION TITLE
capacity credit scenarios

### DISCUSSION THREAD

**Discussion by ssattler** [NREL/ReEDS-2.0#178] [Q&A: Analysis]
I would like to run ReEDS and change the capacity credit fossil generators receive. Trieu suggested the we can derate the amount of fossil that counts towards the planning reserves in the constraints. Could you point to where I should be making these changes?  I'll also be varying the capacity credit contribution by year, so I'd be applying a different % reduction over time. Thanks!

**Comment by wesleyjcole** [comment]
`eq_reserve_margin` is whre the planning reserve is enforced in the model.  Fossil generators are included in the first section of the equation:

```
*[plus] sum of all non-rsc and non-storage capacity
    + sum{(i,v)$[valcap(i,v,r,t)$(not vre(i))$(not hydro(i))$(not storage(i))$(not dr(i))$(not consume(i))$(not forced_retire(i,r,t))],
          CAP(i,v,r,t)
          * (1 + ccseason_cap_frac_delta(i,v,r,ccseason,t))
         }
```

You could make a new parameter called `derate(i,t)` and set it to 1 for everything, then change some of the technologies to have values less than one in certain years.  For example:

```
derate(i,t)$valcap_i(i) = 1 ;
* assign gas-cc units to have a derate value of 0.9 starting in 2030
derate(i,t)$[(yeart(t)>=2030)$gas_cc(i)] = 0.9 ;
```

Or you could just create your own input table and read it in.

Then you could just add `derate` to the equation:

```
*[plus] sum of all non-rsc and non-storage capacity
    + sum{(i,v)$[valcap(i,v,r,t)$(not vre(i))$(not hydro(i))$(not storage(i))$(not dr(i))$(not consume(i))$(not forced_retire(i,r,t))],
          CAP(i,v,r,t) * derate(i,t)
          * (1 + ccseason_cap_frac_delta(i,v,r,ccseason,t))
         }
```
